lib/index.js

```diff
@@ -187,7 +187,7 @@ module.exports = function(options) {
 	return {
 		visitor : {
 			JSXElement: {
-				enter(path, scope) {
```

Comment:
In all the examples I've seen, this has worked, but the parent file has been an ES6 one. Seems to work just fine once you change it from an inline function into a normal `key: value` though.